qujax/version.py

```diff
@@ -1 +1 @@
-__version__ = '0.2.8'
+__version__ = '0.2.9'
```

Comment:
Is this still compatible with the old API? Don't looks like that to me?

Reply:
all previous `qujax` code should still work

we just have access to a new `qujax.apply_gate` function